activesupport/lib/active_support/core_ext/array/access.rb

```diff
@@ -49,6 +49,19 @@ def excluding(*elements)
   end
   alias :without :excluding
 
+  # See <tt>Enumerable#many?</tt>
+  def many?
+    if block_given?
+      cnt = 0
+      any? do |element|
+        cnt += 1 if yield element
+        cnt > 1
+      end
+    else
+      length > 1
+    end
+  end
```

Comment:
The part for a block seems to be a reimplementation of the `Enumerable#many?`. Wonder if we could just call it instead of re-implementing
```suggestion
  def many?
    return super if block_given?
    
    length > 1
  end
```